src/Platform/Bridge/Google/Contract/AssistantMessageNormalizer.php

```diff
@@ -36,7 +36,17 @@ protected function supportsModel(Model $model): bool
     public function normalize(mixed $data, ?string $format = null, array $context = []): array
     {
         return [
-            ['text' => $data->content],
+            array_filter(
+                [
+                    'text' => $data->content,
+                    'functionCall' => ($data->toolCalls[0] ?? null) ? [
+                        'id' => $data->toolCalls[0]->id,
+                        'name' => $data->toolCalls[0]->name,
+                        'args' => $data->toolCalls[0]->arguments ?: new \ArrayObject(),
+                    ] : null,
+                ],
+                static fn ($content) => null !== $content,
+            ),
         ];
```

Comment:
isn't this the same like

```php
$normalized = [];

if (null !== $data->content) {
    $normalized['text'] = $data->content;
}

if (isset($data->toolCalls[0])) {
    $normalized['functionCall'] = [
        'id' => $data->toolCalls[0]->id,
        'name' => $data->toolCalls[0]->name,
        'args' => $data->toolCalls[0]->arguments ?: new \ArrayObject(),
    ];
}

return [$normalized];
```
at least that's more readable to me


could also have a separate `ToolCallNormalizer` but that might be overkill